streaming-pacman/pacman/game/js/scoreboard.js

```diff
@@ -1,5 +1,5 @@
 function loadScoreboardPage() {
-	loadSummaryStats(function(highestScore, usersSet) {
+	loadSummaryStats(function(highestScore, usersSet) { //is highestScore used here at all? I don't think so -JWC
```

Comment:
Not used in this case but the call back should have it defined as that is what will receive.
I guess when we designed it we thought it could be useful. So I'd leave it there for now :)

Reply:
here the reference: https://github.com/confluentinc/demo-scene/blob/ab824ce9f97952125518487a779753cb2549bac7/streaming-pacman/pacman/game/js/shared.js#L42

Reply:
if you define the call back with one param only, it will just receive the value of the first one (highestScore)